src/modules/list.js

```diff
@@ -0,0 +1,97 @@
+/* eslint-disable no-use-before-define */
+/* eslint-disable no-restricted-globals */
+import Cruds from '../cruds/curds.js';
+
+const task = new Cruds();
```

Comment:
I am seeing the same variable name with the export default at task.js. Maybe try to give it a different name.

Reply:
Thanks for notice me. i hope i need to change the same variable with different name.